corehq/reports.py

```diff
@@ -272,7 +272,10 @@ def EDIT_DATA_INTERFACES(domain_obj):
     from corehq.apps.accounting.utils import domain_has_privilege
     reports = [CaseReassignmentInterface]
 
-    if domain_has_privilege(domain_obj.name, privileges.CASE_COPY):
+    if (
+        toggles.COPY_CASES.enabled(domain_obj.name) and
+        domain_has_privilege(domain_obj.name, privileges.CASE_COPY)
```

Comment:
If we are putting this behind a feature flag, should it be a privilege as well?

Reply:
Yeah, it's one of the [requirements](https://dimagi-dev.atlassian.net/browse/SC-2636?focusedCommentId=269252). 

Although, that was before we decided to put it behind a FF. I'll just follow up with the necessary parties about this point.